packages/create-block/lib/prompts.js

```diff
@@ -73,11 +73,40 @@ const category = {
 	choices: [ 'common', 'embed', 'formatting', 'layout', 'widgets' ],
 };
 
+const author = {
+	type: 'input',
+	name: 'author',
+	message:
+		"The author name — this should be a list of wordpress.org user id's:",
```

Comment:
I see it differs a bit between npm and WordPress.org:
https://docs.npmjs.com/files/package.json#people-fields-author-contributors

In Gutenberg I can see the following list:

> Contributors: matveb, joen, karmatosed

We can handle it as a follow-up.